contribution.html

```diff
@@ -110,10 +110,10 @@
             <h2>Documentation</h2>
             <h3>Documentation is one of the main contribution done to the community. Whenever a single learning is documented in a blog lot of people will love to learn from them. And also you can easily write blog post about Firefox, Firefox OS and write MDN articles all it needs in writing knowledge in any language.</h3>
 <div class="btn-group btn-group-justified">
-  <a href="#" class="btn btn-primary" target="_href">Mozillian</a>
-  <a href="#" class="btn btn-primary" target="_href">Etherpad</a>
-  <a href="#" class="btn btn-primary" target="_href">Facbook</a>
-<a href="#" class="btn btn-primary" target="_href">Google+</a>
+  <a href="https://www.mozilla.org/en-US/" class="btn btn-primary" target="_href">Mozillian</a>
+  <a href="https://www.mozilla.org/en-US/" class="btn btn-primary" target="_href">Etherpad</a>
+  <a href="https://www.mozilla.org/en-US/" class="btn btn-primary" target="_href">Facbook</a>
+<a href="https://www.mozilla.org/en-US/" class="btn btn-primary" target="_href">Google+</a>
 </div>

```

Comment:
Sorry I did not know the URL of the respective websites..I didn't know the
g+ URL for mozilla Tamil nadu..
On 19 Feb 2016 4:43 pm, "Tummala Dhanvi (c0mrad3)" <notifications@github.com>
wrote:

> In contribution.html
> <https://github.com/MozillaTN/Weeks-Of-Contirbution/pull/67#discussion_r53448846>
> :
>
> > @@ -110,10 +110,10 @@
> >              <h2>Documentation</h2>
> >              <h3>Documentation is one of the main contribution done to the community. Whenever a single learning is documented in a blog lot of people will love to learn from them. And also you can easily write blog post about Firefox, Firefox OS and write MDN articles all it needs in writing knowledge in any language.</h3>
> >  <div class="btn-group btn-group-justified">
> > -  <a href="#" class="btn btn-primary" target="_href">Mozillian</a>
> > -  <a href="#" class="btn btn-primary" target="_href">Etherpad</a>
> > -  <a href="#" class="btn btn-primary" target="_href">Facbook</a>
> > -<a href="#" class="btn btn-primary" target="_href">Google+</a>
> > +  <a href="https://www.mozilla.org/en-US/" class="btn btn-primary" target="_href">Mozillian</a>
> > +  <a href="https://www.mozilla.org/en-US/" class="btn btn-primary" target="_href">Etherpad</a>
> > +  <a href="https://www.mozilla.org/en-US/" class="btn btn-primary" target="_href">Facbook</a>
> > +<a href="https://www.mozilla.org/en-US/" class="btn btn-primary" target="_href">Google+</a>
> >  </div>
>
> You can't name give the link of mozilla.org for every url
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MozillaTN/Weeks-Of-Contirbution/pull/67/files#r53448846>
> .
>